Lib/idlelib/sidebar.py

```diff
@@ -40,6 +54,15 @@ def get_widget_padding(widget):
     return padx, pady
 
 
+@contextlib.contextmanager
+def temp_enable_text_widget(text):
+    text.configure(state=tk.NORMAL)
+    try:
+        yield
+    finally:
+        text.configure(state=tk.DISABLED)
+
+
 class BaseSideBar:
     """
     The base class for extensions which require a sidebar.
```

Comment:
Docstrings should not start with quotes-only on the first line.  Delete both and 
```suggestion
    "A base class for sidebars using Text."
```

Reply:
Good point.